elastic/kibana
# [Ingest Manager] Do not bumb config revision during config creation

## Summary

Resolve #71774

When we create a agent config with packages inside we we incrementing the revision for each package we install.

This PR Fix that, and the agent config start at revision 1 with with the system package or not.

## UI Example

<img width="1275" alt="Screen Shot 2020-07-17 at 9 05 11 AM" src="https://user-images.githubusercontent.com/1336873/87789685-6ef68080-c80d-11ea-9f7d-e82db16c6bb0.png">


## How to test?

Start from a fresh kibana, the default config should start at revision 1.
Create a new config with the system package the revision should be 1 too.
